code/game/objects/items/storage/boxes.dm

```diff
@@ -609,6 +609,43 @@
 	playsound(loc, "rustle", 50, 1, -5)
 	user.visible_message("<span class='notice'>[user] hugs \the [src].</span>","<span class='notice'>You hug \the [src].</span>")
 
+/////clown box & honkbot assembly
+obj/item/storage/box/clown
+	name = "clown box"
+	desc = "A colorful cardboard box for the clown"
+	icon_state = "clownbox"
+	illustration = null
+	//foldable = null
+
+/obj/item/stack/sheet/cardboard/attackby(obj/item/stamp/clown/I, mob/user, params)
+	if(!istype(I, /obj/item/stamp/clown))
+		..()
+		return
+	to_chat(user, "<span class='notice'>You stamp the cardboard! Its a clown box! Honk!</span>")
+	var/turf/T = get_turf(src)
+	var/obj/item/storage/box/clown/A = new /obj/item/storage/box/clown(T)

```

Comment:
this whole line can be `var/obj/item/storage/box/clown/A = new`, also you can remote the `get_turf` since the item will be put on user's hand